internal/pkg/state/registry/registry.go

```diff
@@ -261,3 +328,47 @@ func (s *Registry) GetOrCreateFrom(manifest ObjectManifest) (ObjectState, error)
 
 	return s.CreateFrom(manifest)
 }
+
+// applyIgnorePattern applies a single ignore pattern, marking the appropriate config or row as ignored.
+func (s *Registry) applyIgnorePattern(pattern string) error {
+	parts := strings.Split(pattern, "/")
+
+	switch len(parts) {
+	case 2:
```

Comment:
this look magical, can we ignore basically any pattern that has any number of parts? 
For example we add new layer/ folder and we need to change the IgnorePattern.